web/src/app/dialogs/FormDialog.js

```diff
@@ -183,31 +192,42 @@ export default class FormDialog extends React.PureComponent {
   }
 
   renderActions = () => {
-    const { alert, confirm, classes, errors, loading, onClose } = this.props
+    const {
+      alert,
+      confirm,
+      classes,
+      errors,
+      loading,
+      onClose,
+      onBack,
+      onNext,
+    } = this.props
 
     if (alert) {
       return (
         <DialogActions>
           <Button color='primary' onClick={onClose} variant='contained'>
-            Okay
+            Done
```

Comment:
Ah, I didn't realize this was previous code and not net-new when I made this change. I would argue that this isn't an alert, so we shouldn't have the "Okay" text, in this case. It makes sense in the case where do have it ("Oops, something went wrong! -> Okay"), whereas here it makes more sense/meets our AC to have it say "Done".

We could prob add a prop `buttonTitle` that overrides any of the conditionals. What do you think?

Reply:
Calling it as an alert will also cause the dialog to stop rendering in full screen, which we likely do not want to do since there is a potential large amount of data we can render in the completion step

Reply:
I went ahead and made the button title change since we can't really have the last step be set as an alert without the fullscreen capability on mobile

Reply:
Yeahh.. This is why I didn't really see `FormDialog` as a fit for this ticket to begin with. It just keeps making that component more and more messy as these custom features come into play. We can add that prop.